mp/src/game/server/momentum/mom_lobby_system.cpp

```diff
@@ -145,29 +87,48 @@ void CMomentumLobbySystem::HandleNewP2PRequestInternal(const SteamNetworkingIden
         return;
     }
 
-    SteamNetworkingMessages()->AcceptSessionWithUser(identity);
+    SteamNetworkingMessages()->AcceptSessionWithUser(pParam->m_identityRemote);
 }
 
-void CMomentumLobbySystem::HandleP2PConnectionFailInternal(const SteamNetworkingIdentity &identity)
+void CMomentumLobbySystem::HandleP2PConnectionFail(SteamNetworkingMessagesSessionFailed_t *pParam)
 {
-    DevLog("%s called\n", __FUNCTION__);
+    if (mom_lobby_debug.GetBool())
+    {
+        const auto iEndReason = pParam->m_info.m_eEndReason;
+        if (iEndReason > k_ESteamNetConnectionEnd_AppException_Min && iEndReason < k_ESteamNetConnectionEnd_AppException_Max)
```

Comment:
These should be inclusive of the min/max

Reply:
Ah yeah I noticed that but it turns out the actual data values are between the boundaries, all except AppException for some reason:

https://github.com/momentum-mod/game/blob/e6819159e8d9e74f6aa0219adde5e8e808f8b37b/mp/src/public/steam/steamnetworkingtypes.h#L408-L610

I'll correct them to be inclusive.

Reply:
Resolved in https://github.com/momentum-mod/game/pull/1000/commits/46f958e915faf501471ac9c7f0a92969c83df11c